github/install.go

```diff
@@ -0,0 +1,232 @@
+package github
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/dlclark/regexp2"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var GithubRegex = regexp.MustCompile(`^https?://(?:www\.)?github\.com/([^/]+/[^/]+)`)
```

Comment:
Would be great to make this regex more specific - handling just the repo differently from a specific file or specific tag would be great.

Reply:
Also, I'd suggest you use lowerCamelCase so that this isn't exported to other packages.

Reply:
> Would be great to make this regex more specific - handling just the repo differently from a specific file or specific tag would be great.

I'm not sure what you mean by this?

> Also, I'd suggest you use lowerCamelCase so that this isn't exported to other packages.

Variables are exported to other packages based on capitalization...!?  
(I'd never written a line of Go before this PR)

Reply:
> I'm not sure what you mean by this?

I mean extracting the file name / tag from a more specific url (e.g. https://github.com/GTNewHorizons/NotEnoughItems/releases/tag/2.6.22-GTNH), like you mentioned in your previous comment.

> Variables are exported to other packages based on capitalization...!?

Yeahh it's a bit silly! I would not choose Go if I'd started writing packwiz today heh.